src/org/ods/util/Project.groovy

```diff
@@ -483,7 +483,10 @@ class Project {
             "techSpecs": [
                 "DEMO-49"
             ],
-            "tests": []
+            "tests": [
+                "PLTFMDEV-401",
```

Comment:
tests PLTFMDEV-401 and PLTFMDEV-1046 will need references to DEMO-50, too, for referential integrity

Reply:
True. Thanks!